src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const shortTerm = 3;
+  const shorTermDiscount = 20;
+  const longterm = 7;
+  const LongTermDiscount = 50;
```

Comment:
The constant `LongTermDiscount` should be in camelCase with the first letter in lowercase to follow JavaScript naming conventions. Please rename it to `longTermDiscount`.